.husky/pre-commit

```diff
@@ -1,8 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npm audit fix
-
-npm run test:silent
-
 npm run lint
```

Comment:
did this not run in precommit. It looks like there are lint errors?

Reply:
must have committed without adding after running lint fix